app/styles/pages/sunday.scss

```diff
@@ -84,12 +42,57 @@
       }
 
       h2 {
-        margin-top: 0;
         font-family: 'Arapey', serif;
-        font-weight: 400;
         font-style: italic;
+        font-weight: 400;
+        margin-top: 0;
         font-size: 36px;
       }
     }
   }
 }
+
+@media only screen and (min-width : $mobile-break) {
+  .bulletin-contents {
+    section.announcements {
+      background-color: #FDF2FD;
+    }
+
+    section.service-order {
+      text-align: center;
+    }
+  }
+}
+
+@media only screen and (min-width : $tablet-portrait-break) {
+  .cover {
+    padding-left: 42px;
+    text-align: left;
+
+    h1 {
+      font-size: 60px;
+      line-height: 50px;
+
+      small {
+        font-size: 24px;
+      }
+    }
+  }
+
+  .bulletin-contents {
+    .inner:before {
+      background-color: #FDF2FD;
```

Comment:
Color literals like `#FDF2FD` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FDF2FD` should be written as `#fdf2fd`